frontend/locales/es.ts

```diff
@@ -0,0 +1,868 @@
+                emailSub: 'El correo electrónico de la cuenta de PayPal donde deben llegar las donaciones',
+                defaultAmount: 'Cantidad por defecto',
+                defaultAmountSub: 'La cantidad por defecto preseleccionada',
+                oneTimeAmounts: 'Cantidades de una vez',
+                oneTimeAmountsSub: 'Las opciones que quieres dar a usuarios que quieren donar solamente una vez. Los usuarios siempre pueden añadir otras cantidades',
+                monthlyAmounts: 'Cantidades mensuales',
+                monthlyAmountsSub: 'Las opciones que los usuarios tiene para donaciones mensuales. Los usuarios siempre pueden añadir otras cantidades',
+            },
+            error: {
+                invalidFile: 'El tipo de archivo {0} es inválido',
+                noFile: 'No has enviado un archivo',
+                members: {
+                    invalidUser: '{0} no es un usuario válido',
+                    alreadyInvited: '{0} ya está invitado al proyecto',
+                    notMember: '{0} no es un miembro del proyecto y por ello no puedes editar su rol',
+                    invalidRole: '{0} no puede ser añadido/eliminado del proyecto',
+                },
+            },
+            success: {
+                changedIcon: 'Se ha cambiado el icono del proyecto exitosamente',
+                resetIcon: 'Se ha reseteado el icono del proyecto exitosamente',
+                rename: 'Se ha cambiado el nombre del proyecto a {0} exitosamente',
+                softDelete: 'Has borrado este proyecto',
+                hardDelete: 'Has borrado este proyecto totalmente',
+            },
+            tabs: {
+                general: 'General',
+                optional: 'Opcional',
+                management: 'Administración',
+                donation: 'Donación',
+            },
+        },
+        discuss: {
+            login: 'Inicia sesion',
+            toReply: 'para responder a este conversación',
+            noTopic: 'No hay conversación en este proyecto',
+            send: '¡Respuesta publicada!',
+        },
+    },
+    page: {
+        plural: 'Páginas',
+        new: {
+            title: 'Crea una nueva página',
+            error: {
+                minLength: 'Los contenidos de la página son demasiado cortos',
+                maxLength: 'Los contenidos de la página son demasiado largos',
+                duplicateName: 'Ya hay una página con ese nombre',
+                invalidName: 'Nombre inválido',
+                name: {
+                    maxLength: 'El nombre de la página es demasiado largo',
+                    minLength: 'El nombre de la página es demasiado corto',
+                    invalidChars: 'El nombre de la página contiene caracteres inválidos',
+                },
+                save: 'No se ha podido guardar la página',
+            },
+            name: 'Nombre de la página',
+            parent: 'Página Superior (opcional)', //TODO: to be checked
```

Comment:
Maybe `Página Primaria (opcional)`?

Reply:
Could be, but I’m not sure where that is used, so it could be in nested pages, and it that case there could be like a superior page of the superior page, or there could just be a parent page, so I think I’ll ask where that is exactly used